rasa_original.py

```diff
@@ -0,0 +1,82 @@
+import aiohttp
+import asyncio
+import async_timeout
+from flask import Flask, request, abort
+
+
+loop = asyncio.new_event_loop()
+app = Flask(__name__)
+
+
+async def incoming_2_outgoing_payload(payload):
+    """Conveert json to xml."""
+    from decoder import Dict2XML
+    # need to convert int keys to str, dict2xml can't handle int values
+    return Dict2XML().parse({
+        "vacation": {k: str(v) for k, v in payload.items()}})
+
+
+async def fetch(url, payload):
+    print("Triggering url {}".format(url))
+    body = incoming_2_outgoing_payload(payload)
```

Comment:
Wait for the coroutine to return results

```suggestion
    body = await incoming_2_outgoing_payload(payload)
```